### PR TITLE
[EDU-3467] feat: add secure menu

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new secureMenu.
* Modified other i18n files to add the new menu.

[EDU-3467]: https://aziontech.atlassian.net/browse/EDU-3467?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by robsongajunior — APPROVED** [review]

File: src/i18n/en/secureMenu.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            { text: 'Manage bots', header: true, anchor: true, type: 'learn', slug: '/documentation/products/guides/secure/manage-bots/', key: 'manageBots', hasLabel: 'menu.secureAdvanced' },
+            { text: 'Activate DDoS', header: true, anchor: true, type: 'learn', slug: '/documentation/products/guides/secure/activate-ddos/', key: 'activateDdos' },
+            { text: 'Create WAF rule set', header: true, anchor: true, type: 'learn', slug: '/documentation/products/guides/secure/create-waf-rule-set/', key: 'wafRuleSet' },
+            { text: 'Tune WAF', header: true, anchor: true, type: 'learn', slug: '/documentation/products/guides/secure/tune-waf/', key: 'tuneWaf' },
+            { text: 'Block Tor networks', header: true, anchor: true, type: 'learn', slug: '/documentation/products/guides/secure/block-tor-networks/', key: 'blockTor' },
+        ]
+    },
+
+
+    { text: 'Select ciphers', header: true, anchor: true, type: 'learn', slug: '/documentation/products/guides/secure/ciphers/', key: 'ciphers', hasLabel: 'menu.secureTransportLayerSecurity' },
```
Comment (bru-andrade) [inline src/i18n/en/secureMenu.ts:44]:
```suggestion
    { text: 'Select TLS ciphers', header: true, anchor: true, type: 'learn', slug: '/documentation/products/guides/secure/ciphers/', key: 'ciphers', hasLabel: 'menu.secureTransportLayerSecurity' },
```

File: src/i18n/pt-br/secureMenu.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            { text: 'Gerencie bots', header: true, anchor: true, type: 'learn', slug: '/documentacao/produtos/guias/secure/gerenciar-bots/', key: 'manageBots', hasLabel: 'menu.secureAdvanced' },
+            { text: 'Ative proteção contra DDoS', header: true, anchor: true, type: 'learn', slug: '/documentacao/produtos/guias/secure/ativar-ddos/', key: 'activateDdos' },
+            { text: 'Adicione uma rule set de WAF', header: true, anchor: true, type: 'learn', slug: '/documentacao/produtos/guias/secure/criar-waf-rule-set/', key: 'wafRuleSet' },
+            { text: 'Tune WAF', header: true, anchor: true, type: 'learn', slug: '/documentacao/produtos/guias/secure/tune-waf/', key: 'tuneWaf' },
+            { text: 'Bloqueie redes Tor', header: true, anchor: true, type: 'learn', slug: '/documentacao/produtos/guias/secure/bloquear-redes-tor/', key: 'blockTor' },
+        ]
+    },
+
+
+    { text: 'Selecione cifras', header: true, anchor: true, type: 'learn', slug: '/documentacao/produtos/guias/secure/cifras/', key: 'ciphers', hasLabel: 'menu.secureTransportLayerSecurity' },
```
Comment (bru-andrade) [inline src/i18n/pt-br/secureMenu.ts:44]:
```suggestion
    { text: 'Selecione cifras TLS', header: true, anchor: true, type: 'learn', slug: '/documentacao/produtos/guias/secure/cifras/', key: 'ciphers', hasLabel: 'menu.secureTransportLayerSecurity' },
```

**Review by bru-andrade — APPROVED** [review]

File: src/i18n/en/secureMenu.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    { text: 'Configure mTLS', header: true, anchor: true, type: 'learn', slug: '/documentation/products/guides/secure/mtls/', key: 'mtls' },
+    { text: 'Manage Digital Certificates', header: true, anchor: true, type: 'learn', slug: '/documentation/products/guides/secure/digital-certificates/', key: 'digitalCertificates' },
+
+
+    { text: 'Edge Functions', header: true, anchor: true, type: 'learn', slug: '/documentation/products/secure/automate/functions/', key: 'automateEdgeFunctions', hasLabel: 'menu.secureAutomate' },
+    { text: 'SDK', header: true, anchor: true, type: 'learn', slug: '/documentation/products/secure/automate/sdk/', key: 'automateSdk' },
+    { text: 'Terraform', header: true, anchor: true, type: 'learn', slug: '/documentation/products/secure/automate/terraform/', key: 'automateTerraform' },
+    { text: 'Protect origin', header: true, anchor: true, type: 'learn', slug: '/documentation/products/secure/automate/origin-shield/', key: 'automateOriginShield' },
```
Comment (MarianaAguilera) [inline src/i18n/en/secureMenu.ts:52]: Protect origins or Protect an origin?

File: src/i18n/pt-br/secureMenu.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    { text: 'Configure mTLS', header: true, anchor: true, type: 'learn', slug: '/documentacao/produtos/guias/secure/mtls/', key: 'mtls' },
+    { text: 'Gerencie certificados digitais', header: true, anchor: true, type: 'learn', slug: '/documentacao/produtos/guias/secure/certificado-digital', key: 'digitalCertificates' },
+
+
+    { text: 'Edge Functions', header: true, anchor: true, type: 'learn', slug: '/documentacao/produtos/secure/automarizar/functions/', key: 'automateEdgeFunctions', hasLabel: 'menu.secureAutomate' },
+    { text: 'SDK', header: true, anchor: true, type: 'learn', slug: '/documentacao/produtos/secure/automarizar/sdk/', key: 'automateSdk' },
+    { text: 'Terraform', header: true, anchor: true, type: 'learn', slug: '/documentacao/produtos/secure/automarizar/terraform/', key: 'automateTerraform' },
+    { text: 'Proteja origem', header: true, anchor: true, type: 'learn', slug: '/documentacao/produtos/secure/automarizar/origin-shield/', key: 'automateOriginShield' },
```
Comment (MarianaAguilera) [inline src/i18n/pt-br/secureMenu.ts:52]: same

**Review by MarianaAguilera — APPROVED** [review]
Deixei uma sugestão, para sua consideração

**Pull request merged**